web/models/stable_diffusion/main.py

```diff
@@ -10,6 +10,24 @@
 from random import randint
 import numpy as np
 import time
+import sys
+
+
+# Helper function to profile the vulkan device.
+def start_profiling(file_path="foo.rdc", profiling_mode="queue"):
```

Comment:
We should check whether anyone is using this feature and if not, drop it. Another option would be to make it available for all models via SharkRunner.

Reply:
yeah, we can discuss on this and put it at the right place.